.github/workflows/pytest-dependency.yml

```
@@ -25,7 +25,7 @@ jobs:
     - name: Install specific out-dated version of dependencies
       # Update the package requirements when changing minimum dependency versions
       # Please also add a section "Dependency changes" to the release notes
-      run: pip install pandas==1.1.1 numpy==1.19.0 matplotlib==3.5.0 iam-units==2020.4.21 xlrd==2.0
+      run: pip install pandas==1.1.1 numpy==1.19.0 matplotlib==3.5.0 iam-units==2020.4.21 xlrd==2.0 pint==0.17
```

This is not consistent with the specification and is actively overwritten by the installation in the following, see the [Actions log](https://github.com/IAMconsortium/pyam/runs/5448754850?check_suite_focus=true)
```suggestion
      run: pip install pandas==1.1.1 numpy==1.19.0 matplotlib==3.5.0 iam-units==2020.4.21 xlrd==2.0 pint==0.18
```

---

Ah, then I misunderstood the lagacy-dependency test.